axelrod/strategies/_strategies.py

```diff
@@ -22,6 +22,7 @@
     DefectorHunter, CooperatorHunter, CycleHunter, AlternatorHunter,
     MathConstantHunter, RandomHunter, EventualCycleHunter)
 from .inverse import Inverse
+from .lookerup import EvolvedLookerUp
```

Comment:
Did you only want EvolvedLookerUp to go into the default set of strategies, or also LookerUp as well?